README.md

```diff
@@ -1,193 +1,94 @@
+* Letting you create, edit, and delete Zowe CLI `zosmf` compatible profiles.
+* Letting you use the Secure Credential Store plug-in to store your credentials securely in the settings.
 
-**Note:** The Zowe Explorer is powered by [Zowe CLI](https://zowe.org/home/). The extension demonstrates the potential for plug-ins powered by Zowe.
-
-**Tips:**
-
-* For information about how to install the extension from a `VSIX` file and run system tests on the extension, see the [Developer README](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README.md).
-* For information about how to develop for Eclipse Theia, see [Theia README](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README-Theia.md).
-* For information about how to create a VSCode extension for Zowe Explorer (a Zowe Explorer extension) see [VSCode extensions for Zowe Explorer](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README-Extending.md).
-* For questions and general guidance, please visit #zowe-explorer channel on [Slack](https://openmainframeproject.slack.com/)
+Join the **#zowe-explorer** channel on [Slack](https://openmainframeproject.slack.com/) to stay in touch with the Zowe community.
 
 ## Contents
 
-* [Software Requirements](#software-requirements)
-* [Create a Zowe CLI z/OSMF profile](#create-a-zowe-cli-zosmf-profile)
-* [Secure Credential Store](#secure-credential-store)
-* [Usage tips](#usage-tips)
-* [Sample use cases](#sample-use-cases)
+* [What's new in Zowe Explorer 1.6.0](#what's-new-in-zowe-explorer-1.6.0)
+* [Prerequisites](#prerequisites)
+* [Getting Started](#getting-started)
+* [Credentials Security](#credentials-security)
+* [Usage Tips](#usage-tips)
+* [Extending Zowe Explorer](#extending-zowe-explorer)
 
-## Software Requirements
+## What's new in Zowe Explorer 1.6.0
 
-Ensure that you meet the following prerequisites before using the extension:
+New features:
 
-* Installed [Node.js](https://nodejs.org/en/download/) v8.0 or later.
-* Configured TSO/E address space services, z/OS data set and file REST interface, and z/OS jobs REST interface. For more information, see [z/OS Requirements](https://docs.zowe.org/stable/user-guide/systemrequirements-zosmf.html#z-os-requirements).
-* Zowe CLI `zosmf` profile.
+* Create profiles with different, other than `zosmf`, profile types. For example, if you have the FTP plug-in, you can create a profile of the FTP type.
+* Improved user experience around adding new profiles. If the quick pick list does not contain any profiles, you will be prompted to create a new profile instead of picking the option to create a new profile from the empty list.
 
-**Notes:**
+Fixed bugs:
 
-* You can use your existing Zowe CLI `zosmf` profiles that are created with the Zowe CLI v.2.0.0 or later.
-* Zowe CLI `zosmf` profiles that are created in Zowe Explorer can be interchangeably used in the Zowe CLI.
+* Fixed persistent data after the recall functionality.
```

Comment:
Maybe it should say instead "Fixed removal of items from persistent recent files list"

Or you could remove this point altogether, it was a fix that came up because of delete connection, so it's kind of covered in the next point.